runner/Dockerfile

```diff
@@ -10,14 +10,13 @@ ENV TERM="dumb"
 # These should be cached
 WORKDIR /usr/src/ferris-elf
 COPY Cargo.toml Cargo.toml
+COPY benches/bench.rs benches/bench.rs
 COPY src/main.rs src/main.rs
-COPY src/placeholder.rs src/lib.rs
```

Comment:
The problem with this Dockerfile is that it was building the working directory a specific way in order to deal with certain hypothetical issues. However, when we re-wrote the runner, the Rust code no longer matched the assumptions the Dockerfile was making, so the Dockerfile broke.

There's probably a much cleaner way of doing this, but iirc, we won't be keeping this setup anyway, so I wasn't going to worry too hard about it.